gulp/tasks/scripts.js

```diff
@@ -208,24 +187,22 @@ function scriptsApps() {
   }
   return singleStream;
 }
-
-gulp.task( 'scripts:apps', scriptsApps );
-gulp.task( 'scripts:external', scriptsExternal );
-gulp.task( 'scripts:modern', scriptsModern );
-gulp.task( 'scripts:polyfill', scriptsPolyfill );
-gulp.task( 'scripts:admin', scriptsAdmin );
-
 gulp.task( 'scripts:ondemand:header', scriptsOnDemandHeader );
 gulp.task( 'scripts:ondemand:footer', scriptsOnDemandFooter );
-gulp.task( 'scripts:ondemand:nonresponsive', scriptsNonResponsive );
+
 gulp.task( 'scripts:ondemand',
   gulp.parallel(
     'scripts:ondemand:header',
-    'scripts:ondemand:footer',
-    'scripts:ondemand:nonresponsive'
+    'scripts:ondemand:footer'
   )
 );
 
+gulp.task( 'scripts:apps', scriptsApps );
+gulp.task( 'scripts:external', scriptsExternal );
+gulp.task( 'scripts:modern', scriptsModern );
+gulp.task( 'scripts:polyfill', scriptsPolyfill );
+gulp.task( 'scripts:admin', scriptsAdmin );
+
```

Comment:
Why move these down here?

Reply:
Just a consequence of re-adding the on-demand scripts in a different place. I moved these back, though it shouldn't matter.